File: 1-adhock/2-bandingkan-angka/solution.js
function bandingkanAngka(A, B) {
  if (B > A) {
    return true
  } else if (A == B) {
    return -1
  }
  return false
}

// Driver Code
console.log(bandingkanAngka(5, 8)) // True
console.log(bandingkanAngka(5, 3)) // False
console.log(bandingkanAngka(4, 4)) // -1
console.log(bandingkanAngka(3, 3)) // -1
console.log(bandingkanAngka(17, 2)) // False

File: 1-adhock/2-bandingkan-angka/solution.go
package main

import "fmt"

func bandingkanAngka(a int, b int) interface{} {
	if b > a {
		return true
	} else if a == b {
		return -1
	}
	return false
}

func main() {
	// Driver Code
	fmt.Println(bandingkanAngka(5, 8))  // True
	fmt.Println(bandingkanAngka(5, 3))  // False
	fmt.Println(bandingkanAngka(4, 4))  // -1
	fmt.Println(bandingkanAngka(3, 3))  // -1
	fmt.Println(bandingkanAngka(17, 2)) // False
}
